app/javascript/app/components/emission-pathways-table-menu/emission-pathways-table-menu-component.jsx

```diff
@@ -8,12 +8,31 @@ import anchorNavLightTheme from 'styles/themes/anchor-nav/anchor-nav-light.scss'
 import styles from './emission-pathways-table-menu-styles.scss';
 
 class EmissionPathwaysTableMenu extends PureComponent {
+  renderSingleTitle() {
```

Comment:
I dont think this is needed. If you are rendering a single title, then just render a string in the component.